app/common/containers/pages/SignInPage/redux.js

```diff
@@ -19,15 +19,11 @@ dispatch(createSessionToken({
     const { message } = action.payload.response.error;
     if (message === 'User blocked.') {
       throw new SubmissionError({
-        email: { user_blocked: true },
+        email: { accountPasswordMismatch: true },
```

Comment:
why do we throw account mismatch error if user is blocked?

Reply:
Mistype, thx)